### DISCUSSION TITLE
Welcome to Open-ST Discussions!

### DISCUSSION THREAD

**Discussion by danilexn** [rajewsky-lab/openst#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by velmeshevlab** [comment]
Thanks so much for a great protocol paper and making everything open-source and so easy to follow! I have a question about sequencing machine model/flow cell: can NextSeq 2000 be used for Open-ST?

**Reply by lmschott** [reply]
Hi, 
Thanks for the kind words! I believe in theory the NextSeq2000 can also be used for Open-ST, but we have never tried it so I have no tips to give. The NextSeq2000 is also patterned and actually has a higher cluster density (~4900 vs 2900 K/mm^2 of the NovaSeq6000). I have tried opening a used NextSeq2000 flow cell before and know that the glue that holds it together is much stronger, so I would suggest practicing that before giving it a try.

**Comment by Y-SASAGAWA** [comment]
Thank you for publishing detailed information on Open-ST. 

I have a question on one of the product you use.
The following products are used in the protocol of the paper. 
```
16-Well ProChamber Microarray System	
Grace Bio-Labs	
Cat#645508
```

I tried to purchase this product but could not find the catalogue number on the GraceBio-Labs home page (https://gracebio.com/). 
If you know the corresponding URL, could you please let me know? 

Also, if you know of an alternative product name and its part number, I would be grateful if you could let me know.

Thank you in advance.

**Reply by lmschott** [reply]
Hi, 
It seems that Grace Biolabs has stopped selling this product. 

As an alternative we also use: 
Grace Bio-Labs ProPlate® microarray system, slide module (GBL204862-2EA), https://www.sigmaaldrich.com/DE/en/product/sigma/gbl204862 
or 
12 Well Chamber, removable (Cat.No:81201), https://ibidi.com/chamber-slides/46-12-well-chamber-removable.html 

Either works with the ~3x4 mm pieces and 100 uL reaction volume, and we seal them with sticky tape. 
We reuse the Grace-biolabs product (cleaning with ethanol and RNase away). The ibidi chamber becomes leaky at some point, so I don't like to reuse it. 
With these products I have (very rarely) observed some leaking and they are more difficult to seal efficiently. Thus, I like to space out my capture areas and make a humid chamber for my overnight incubation.
Let us know if you have other suggestions :)

**Reply by Y-SASAGAWA** [reply]
Thank you for your introduction and detailed explanation of alternative products.

I will try these two products first.

It seems that there are various slide chambers available from other companies.

If I find something good, I will give feedback here.

**Reply by lmschott** [reply]
Thank you!

**Comment by Rin-Kawagoe** [comment]
Thank you for publishing the detailed information on Open-ST.
I have two questions regarding the protocol:

1. Could you please share the manufacturer and product number of the plate seal used for capture area labeling?  I'm concerned that a plastic seal might dissolve in 100% methanol during the fixation step.
2. Are the Hematoxylin and Eosin used in Open-ST certified as RNase/DNase-free?

**Reply by lmschott** [reply]
Hi! 
1) I use SealPlate foil from Excel Scientific (cat# Z369667). You can buy it via Sigma-Aldrich for example. It sticks well throughout, but can also be removed well without leaving any residue.
2) They are not certified as nuclease-free.

**Reply by Rin-Kawagoe** [reply]
Thank you for your response.
I am concerned that the plate seal might dissolve, so I would appreciate it if you could provide more details about the methanol fixation process.
For example, is the procedure carried out inside the cryostat, or in a fume hood while keeping the sample cooled with dry ice?
Additionally, it would be helpful to know the type of tube (e.g., 1.5 mL, 5 mL, etc.) used to hold the methanol.

**Reply by lmschott** [reply]
Sure! We fix in methanol for 30 min at -20°C. For each sample I put 1 mL 100% methanol in a 2 mL roundbottom tube and let that pre-cool in a -20°C freezer. 
For safety reasons, I would transfer the samples (cryosection on the capture area stuck to sticky tape) on dry ice to a fume hood and then place them in the pre-cooled methanol. Once the sample is in the methanol, I place the closed tubes back in the -20°C. 
Hope this helps!

**Reply by Rin-Kawagoe** [reply]
Thank you for your detailed explanation.
I will try it as you described.
